packages/waku/src/lib/middleware/rsc/worker-impl.ts

```diff
@@ -18,6 +18,7 @@ import {
   defineEntries,
   runWithAsyncLocalStorage as runWithAsyncLocalStorageOrig,
 } from '../../../server.js';
+import { normalizePath } from 'vite';
```

Comment:
Same here again. If we need such a functionality in production, we need to create one for ourselves.